src/services/webSocketClient.ts

```diff
@@ -8,13 +8,20 @@ interface IAddMessageListenerOptions {
   debug?: string;
 }
 
-const webSocketClient = (url: string) => {
+export const createWebSocketClient = (url: string) => {
```

Comment:
WebSocketClient was slightly refactored to make sure new Bifrost clients are created when subscribing a new log and client will be closed when leaving the page. This will make sure previous logs will not impact other logs, because we have no way to unsubscribe the WS message from our Bifrost server, those big builds logs can override other logs if we share the same client.